admin_interface/templates/admin/base_site.html

```diff
@@ -40,11 +35,10 @@
 {% block blockbots %}
 {{ block.super }}
 {% get_admin_interface_theme as theme %}
-{% get_admin_interface_version as version %}
 {# https://github.com/elky/django-flat-responsive#important-note #}
 <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1.0, maximum-scale=1.0">
-<link rel="stylesheet" type="text/css" href="{% static 'admin/css/responsive.css' %}?v={{ version }}">
-<link rel="stylesheet" type="text/css" href="{% static 'admin/css/responsive_rtl.css' %}?v={{ version }}" />
+<link rel="stylesheet" type="text/css" href="{% static 'admin/css/responsive.css' %}">
+<link rel="stylesheet" type="text/css" href="{% static 'admin/css/responsive_rtl.css' %}" />
```

Comment:
```suggestion
<link rel="stylesheet" type="text/css" href="{% static 'admin/css/responsive_rtl.css' %}">
```
(proper HTML self-closing tag)